caddy/https/setup.go

```diff
@@ -220,6 +228,11 @@ func loadCertsInDir(c *setup.Controller, dir string) error {
 // port to 443 if not already set, TLS is enabled, TLS is manual, and the host
 // does not equal localhost.
 func setDefaultTLSParams(c *server.Config) {
+	// If no KeyType provided, use EC384 as preferred KeyType
```

Comment:
I'm gonna be really nit-picky for just a sec, I know, but let's just delete this comment entirely. It's out of date and doesn't tell us something non-obvious.